features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Permission.js

```diff
@@ -16,55 +16,305 @@
+            title: 'Scopes',
+            dataIndex: 'name',
+            key: 'name',
+            render: text => <a href="#">{text}</a>,
+        }, {
+            title: '',
+            key: 'delete',
+            render: (text, record) => (
+                <span>
+                    <a href="#">Delete</a>
+                </span>
+            ),
+        }];
+
+        const dataOfScopes = [{
+            key: '1',
+            name: 'John Brown',
+
+        }, {
+            key: '2',
+            name: 'Jim Green',
+
+        }];
+
+        const data = [{
+            key: '1',
+            name: 'Engineering',
+            age: 'Yes',
+            address: 'New York No. 1 Lake Park',
+            update: 'update'
+        }, {
+            key: '2',
+            name: 'Sales',
+            age: 42,
+            address: 'New York No. 1 Lake Park',
+            update: 'update'
+        }, {
+            key: '3',
+            name: 'QA',
+            age: 32,
+            address: 'New York No. 1 Lake Park',
+            update: 'update'
+        }];
+
+
+        const api = this.state.api;
+        if (this.state.notFound) {
+            return <ResourceNotFound/>
+        }
+        if (!this.state.api) {
+            return <Loading/>
+        }
+
 
         return (
-            <div>
-                <div>
-                    <div className="wrapper wrapper-content">
-                        <h2> API Name : {this.state.api.name} </h2>
-
-                        <div className="divTable">
-                            <div className="divTableBody">
-                                <div className="gutter-example">
-                                    <Row gutter={16}>
-                                        <Col className="gutter-row" span={6}>
-                                            <div className="gutter-box">Group Name</div>
-                                        </Col>
-                                        <Col className="gutter-row" span={6}>
-                                            <div className="gutter-box">Read</div>
-                                        </Col>
-                                        <Col className="gutter-row" span={6}>
-                                            <div className="gutter-box">Update</div>
-                                        </Col>
-                                        <Col className="gutter-row" span={6}>
-                                            <div className="gutter-box">Delete</div>
-                                        </Col>
-                                    </Row>
-                                </div>
 
+            <div>
+                <Row type="flex" justify="left">
+                    <Col span={4}>
+                        <Card bodyStyle={{padding: 5}}>
+                            <div className="custom-image">
+                                <img alt="API thumb" width="100%" src="/publisher/public/images/api/api-default.png"/>
```

Comment:
Can you remove this inline styling (width="100%") and move to a css file. There are few more in the PR as well.